### PR TITLE
Fill out service.yml for staging skyline pipeline

### PR DESCRIPTION
**Pull request by pinkertonpg**
## What
Setup avro-schema-registry for running our build/test/deploy pipeline in jenkins/kubernetes:
- Remove the $PORT in the Dockerfile. This is unneeded and adds extra config in several places
- Add a `staging` section to `database.yml`. This _can_ be replaced with a `DATABASE_URL`, I'm not sure if we have a preference here.
- Fill out `service.yml` with all of the required fields to build asr in jenkins (docker image), test that image with rspec, push the image to artifactory, and then kick off a deploy into the staging kubernetes cluster.

Once this in master, any commits to master will automatically be deployed to the staging kube cluster (and a feature branch can also be deployed by adding `[push image]` a the commit message).

## Why
We're moving into kubernetes, and asr is the simplest app to start with.

Currently running in staging: https://asr.staging-ezcater.com/

### CODE REVIEW

File: service.yml [hunk 1]
```diff
@@ -1 +1,40 @@
...
+        POSTGRES_USER: ezcater
+        PORT: 3000
+  services:
+    - name: postgres
+      image: postgres:9.6
+      environment:
+        POSTGRES_USER: ezcater
+        POSTGRES_DB: avro-schema-registry_test
+  steps:
+    - run:
```
Comment (tjwp) [inline service.yml]: Should steps like running rubocop be included too?

File: service.yml [hunk 2]
```diff
@@ -1 +1,40 @@
...
+  primary:
+    - name: avro-schema-registry
+      build: .
+      environment:
+        POSTGRES_HOST: localhost
+        POSTGRES_USER: ezcater
+        PORT: 3000
+  services:
+    - name: postgres
+      image: postgres:9.6
```
Comment (tjwp) [inline service.yml]: Should patch version be specified here? In development we've seen differences in the schema dump between 9.6.6 and 9.6.8.

Comment (pinkertonpg) [reply service.yml]: Sure! I'll update this and the docker-compose to stick to 9.6.8.

File: Dockerfile [hunk 3]
```diff
@@ -13,4 +13,4 @@ RUN bundle install
 
 ADD . /usr/src/myapp
 
-CMD rails server -p $PORT -b 0.0.0.0
+CMD rails server -p 3000 -b 0.0.0.0
```
Comment (zoso10) [inline Dockerfile]: do we need an `EXPOSE` for this port?

File: service.yml [hunk 4]
```diff
@@ -1 +1,43 @@
...
+  services:
+    - name: postgres
+      image: postgres:9.6.8
+      environment:
+        POSTGRES_USER: ezcater
+        POSTGRES_DB: avro-schema-registry_test
+  steps:
+    - run:
+        test: rubocop
+        command: rubocop
```
Comment (zoso10) [inline service.yml]: should we `bundle exec` it?

Comment (pinkertonpg) [reply service.yml]: Probably!

File: Dockerfile (hunk 3 above)
Comment (pinkertonpg) [reply Dockerfile]: It's not strictly required. I imagine we'd only be running this via docker-compose, or kubernetes, in which case they handle linking other containers to this one. It's definitely helpful for documentation though, so I've gone ahead and added it.

Comment (zoso10) [reply Dockerfile]: gotcha, it does have a root "homepage" which ive navigated to locally a couple times to check it, but aside from that it should only be hit from other containers

**Pull request merged**